src/creds/creds.c

```diff
@@ -651,36 +652,36 @@ static int verb_setup(int argc, char **argv, void *userdata) {
         return EXIT_SUCCESS;
 }
 
-static int verb_has_tpm2(int argc, char **argv, void *userdata) {
-        Tpm2Support s;
+static int chainload_analyze(int argc, char **argv, void *userdata) {
```

Comment:
Hmm, instead we should extract `verb_has_tpm2` to `shared/verb-has-tpm2.c` or just place it in `tpm2-util`, similar to how verb-log-control is handled?

Reply:
Yeah, I agree with @YHNdnzj. systemd-analyze may not be installed on the system.

Reply:
Dunno, I am pretty sure wwe should go the other way, and move verb-log-control also to such a chainload scheme. I don't think we want to redudnantly implement everything at various places, but chainloading is much preferable for compat.

And the fact that systemd-analyze is not installed everywhere is not really a problem, this call isn't really part of any clean codepaths, hence systemd-analyze is the *right* place for it, along with the other tpm analysis tool.

Reply:
But the redundancy is minor, no? And I think we should avoid chainloading needlessly.